cmd/osbuild-composer/config.go

```diff
@@ -11,11 +11,12 @@ import (
 )
 
 type ComposerConfigFile struct {
-	Koji      KojiAPIConfig   `toml:"koji"`
-	Worker    WorkerAPIConfig `toml:"worker"`
-	WeldrAPI  WeldrAPIConfig  `toml:"weldr_api"`
-	LogLevel  string          `toml:"log_level"`
-	LogFormat string          `toml:"log_format"`
+	Koji         KojiAPIConfig   `toml:"koji"`
+	Worker       WorkerAPIConfig `toml:"worker"`
+	WeldrAPI     WeldrAPIConfig  `toml:"weldr_api"`
+	SyslogServer string          `toml:"syslog_server" env:"SYSLOGSRVR"`
```

Comment:
I would probably go with `SYSLOG_SERVER`, I don't think we need to save bytes,

Reply:
I agree `SYSLOG_SERVER` is better, I made it this way for consistency with other env vars like `PGUSER` and `PGPASSWORD`

Reply:
PG* etc. are standard postgres variables https://www.postgresql.org/docs/current/libpq-envars.html, so I just used the colloquial ones there.